editoast/src/views/round_trips.rs

```diff
@@ -0,0 +1,105 @@
+use axum::response::IntoResponse;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+use utoipa::openapi::RefOr;
+use utoipa::openapi::Schema;
+
+use crate::error::Result;
+
+crate::routes! {
+    "/round_trips" => {
```

Comment:
Ideally we'd find a better name for this which includes both "one-way" and "round trip", but I haven't found one.

Reply:
Maybe just "trip"?

Reply:
Honestly, I prefer round trips. "association" or "link" have the same problem because it's strange to store non-associates too.

Reply:
Can I mark as resolved @woshilapin @emersion ?